### ISSUE TITLE
form does not have the a row '_token'

### ISSUE THREAD

**Issue by gassan** [symfony/symfony#45224]
### Symfony version(s) affected

5.4.3

### Description

If **framefork.form.csrf_protection** not set, the form row **_token** is not created.
if **framefork.form.csrf_protection** set to null (~) then all is running as expected.

### How to reproduce

Do not set **framefork.form.csrf_protection**

**Comment by ro0NL** [comment]
it's a feature https://github.com/symfony/symfony/blob/4834d9e224b97f946fe8e13b2ba403bbc212c4ad/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L190

**Comment by gassan** [comment]
Look to recipe. the **form** section is not set at all. But **form. csrf_protection** is treated as disabled in this case.

**Comment by stof** [comment]
according to the comment, it should have a smart default.

**Comment by ro0NL** [comment]
https://github.com/symfony/symfony/blob/4834d9e224b97f946fe8e13b2ba403bbc212c4ad/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L495-L498

so perhaps because of full stack install maybe?

**Comment by gassan** [comment]
I thought if some config is woking good with 5.4.2 it must be also correct for 5.4.3.
But ok, I can add suddenly required option **form.csrf_tokren** to my config/framework-bundle.yaml

**Comment by VincentLanglet** [comment]
Not sure if it's fully related, but I just bumped the dependency and got some 
```
The option "csrf_protection" does not exist
```
errors which I didn't have on 5.4.2 versions

**Comment by ro0NL** [comment]
(sorry, i misread the issue being broken between 2 versions)

**Comment by gassan** [comment]
> https://github.com/symfony/symfony/blob/4834d9e224b97f946fe8e13b2ba403bbc212c4ad/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L495-L498
> 
> so perhaps because of full stack install maybe?

No, $config['csrf_protection']['enabled'] is true after this code block.

**Comment by stof** [comment]
I think this is caused by https://github.com/symfony/symfony/pull/44998 changing the order of processing the config.

@Seldaek it looks like you moved the CSRF config to the end because it depends on the session config that you moved. But then, you forgot other configs that depend on the CSRF (at least, form)

**Comment by Seldaek** [comment]
Yeah I was pretty terrified by this change, as it was fairly likely to break something somewhere not covered by tests.

**Comment by gassan** [comment]
moving
https://github.com/symfony/symfony/blob/4834d9e224b97f946fe8e13b2ba403bbc212c4ad/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L346
down solves this bug, but it can probable produce other bugs.

**Comment by hanishsingla** [comment]
Sometimes, you need to disable "_token" for specific form only https://symfony.com/doc/current/security/csrf.html#csrf-protection-in-symfony-forms
